lib/states/statesInMemServerRedis.js

```diff
@@ -76,7 +76,7 @@ class StatesInMemoryServer extends StatesInMemoryFileDB {
     /**
      * Separate Namespace from ID and return both
      * @param idWithNamespace ID or Array of IDs containing a redis namespace and the real ID
-     * @returns {{namespace: (string), id: (string | string[])}} Object with namespace and the
```

Comment:
dito - why is this duplicated btw?

Reply:
why the method is duplicated? Is it really "the same" in objects and states? don't think so

Reply:
nevermind, I got confused somehow ;)